geom.js

```diff
@@ -1,28 +1,76 @@
 class Rectangle {
-  constructor(length, width) {
-    this.length = length;
-    this.width = width;
+  constructor (length, width) {
+    this.length = length
+    this.width = width
+  }
+  isSquare () {
+    if (this.length === this.width) {
```

Comment:
Since the statement `this.length === this.width` evaluates to a `true`/`false` value, we can directly return that result instead of wrapping it in an `if` statement to explicitly return `true` or `false`:

```js
return this.length === this.width
```